spec/integration/network/http/api/indirected_routes_spec.rb

```diff
@@ -2,6 +2,8 @@
 require 'spec_helper'
 require 'puppet/network/http'
 require 'puppet/network/http/api/indirected_routes'
+require 'rack/mock' if Puppet.features.rack?
```

Comment:
Yeah, I had initially put the require down in the describe in https://github.com/puppetlabs/puppet/pull/4875/commits/363d36e232a5be40646d1412bbc958281677a581 but that failed because it either needed to be conditionally required or the require needed to move into the `it` block. I moved it up there based on reviewing things like [the rack specs](https://github.com/puppetlabs/puppet/blob/f50edf20fd963a357f98924efa6000e5ac238e76/spec/unit/network/http/rack_spec.rb#L3). Though I have no strong preference to where it goes.